src/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>To Do List</title>
+    <link
+      rel="stylesheet"
+      href="https://unicons.iconscout.com/release/v4.0.8/css/line.css"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="my-todo"><h1>My To Do List</h1></div>
+      <div class="input-field">
+        <textarea placeholder="Enter your todo"></textarea>
+        <i class="uil uil-notes note-icon"></i>
+      </div>
+
+      <ul class="todoLists"></ul>
+
+      <div class="pending-tasks">
```

Comment:
Kindly remove the unnecessary blank lines in the HTML to maintain the HTML best practices.

Reply:
It is because of the default formatter that I use. But I have already re-formatted my code and solved the issue. Please check my latest commits. 😉